.github/workflows/vale-lint.yml

```diff
@@ -0,0 +1,40 @@
+name: Validate content with Vale
+
+on:
+  pull_request:
+    paths:
+      - 'platform/**'
```

Comment:
Shouldn't this also be limited to markdown and mdx files?
Otherwise a change to the package.json would retrigger this.